packages/react-core/src/demos/Page/Page.md

```diff
@@ -25,6 +27,312 @@ This demonstrates a variety of navigation patterns in the context of a full page
+      </DropdownItem>,
+      <DropdownItem>
+        <HelpIcon /> Help
+      </DropdownItem>
+    ];
+
+    const userDropdownItems = [
+      <DropdownGroup key="group 2">
+        <DropdownItem key="group 2 profile">My profile</DropdownItem>
+        <DropdownItem key="group 2 user" component="button">
+          User management
+        </DropdownItem>
+        <DropdownItem key="group 2 logout">Logout</DropdownItem>
+      </DropdownGroup>
+    ];
+
+    const fullKebabItems = [
+      <DropdownGroup key="group 2">
+        <DropdownItem key="group 2 profile">My profile</DropdownItem>
+        <DropdownItem key="group 2 user" component="button">
+          User management
+        </DropdownItem>
+        <DropdownItem key="group 2 logout">Logout</DropdownItem>
+      </DropdownGroup>,
+      <Divider key="divider" />,
+      <DropdownItem key="kebab-1">
+        <CogIcon /> Settings
+      </DropdownItem>,
+      <DropdownItem key="kebab-2">
+        <HelpIcon /> Help
+      </DropdownItem>
+    ];
+
+    const headerToolbar = (
+      <Toolbar id="toolbar" isFullHeight isStatic>
+        <ToolbarContent>
+          <ToolbarGroup
+            variant="icon-button-group"
+            alignment={{ default: 'alignRight' }}
+            spacer={{ default: 'spacerNone', md: 'spacerMd' }}
+          >
+            <ToolbarItem>
+              <Button aria-label="Notifications" variant={ButtonVariant.plain}>
+                <AttentionBellIcon />
+              </Button>
+            </ToolbarItem>
+            <ToolbarGroup variant="icon-button-group" visibility={{ default: 'hidden', lg: 'visible' }}>
+              <ToolbarItem>
+                <Button aria-label="Settings actions" variant={ButtonVariant.plain}>
+                  <CogIcon />
+                </Button>
+              </ToolbarItem>
+              <ToolbarItem>
+                <Button aria-label="Help actions" variant={ButtonVariant.plain}>
+                  <HelpIcon />
+                </Button>
+              </ToolbarItem>
+            </ToolbarGroup>
+          </ToolbarGroup>
+          <ToolbarItem visibility={{ default: 'hidden', md: 'visible', lg: 'hidden' }}>
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={this.onKebabDropdownSelect}
+              toggle={<KebabToggle onToggle={this.onKebabDropdownToggle} />}
+              isOpen={isKebabDropdownOpen}
+              dropdownItems={kebabDropdownItems}
+            />
+          </ToolbarItem>
+          <ToolbarItem visibility={{ default: 'visible', md: 'hidden', lg: 'hidden', xl: 'hidden', '2xl': 'hidden' }}>
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={this.onFullKebabSelect}
+              toggle={<KebabToggle onToggle={this.onFullKebabToggle} />}
+              isOpen={isFullKebabDropdownOpen}
+              dropdownItems={fullKebabItems}
+            />
+          </ToolbarItem>
```

Comment:
All of the dropdowns/icon buttons go in the same base `icon-button-group` for proper spacing.

```
<ToolbarGroup
  variant="icon-button-group"
  alignment={{ default: 'alignRight' }}
  spacer={{ default: 'spacerNone', md: 'spacerMd' }}
>
  <ToolbarItem>Bell</ToolbarItem>
  <ToolbarItem>Cog</ToolbarItem>
  <ToolbarItem>Help</ToolbarItem>
  <ToolbarItem>Dropdown</ToolbarItem>
  <ToolbarItem>Dropdown</ToolbarItem>
</ToolbarGroup>
```

Then place the user dropdown outside of the `icon-button-group`

```
<ToolbarItem visibility={{ default: 'hidden', md: 'visible' }}>
  <Dropdown
    position="right"
    onSelect={this.onDropdownSelect}
    isOpen={isDropdownOpen}
    toggle={
      <DropdownToggle icon={<Avatar src={imgAvatar} alt="Avatar" />} onToggle={this.onDropdownToggle}>
        John Smith
      </DropdownToggle>
    }
    dropdownItems={userDropdownItems}
  />
</ToolbarItem>
```

Reply:
I actually copied that from `DashboardHeader.js` want me to go ahead and fix it there too? Or since that's not directly related would it be too out of scope for this refactor?